CefSharp.BrowserSubprocess.Core/JavascriptPropertyWrapper.h

```diff
@@ -17,9 +17,9 @@ namespace CefSharp
         JavascriptProperty^ _javascriptProperty;
         int64 _ownerId;
         IBrowserProcess^ _browserProcess;
+    internal:
         //TODO: Strongly type this variable - currently trying to include JavascriptObjectWrapper.h creates a circular reference, so won't compile
         Object^ _javascriptObjectWrapper;
```

Comment:
Making this property internal means that it also needs to be renamed. Prefix with `_` is only for private variables.